KerbalVR_Mod/KerbalVR/InteractionSystem/KerbalVR_Hand.cs

```diff
@@ -112,54 +131,47 @@ public void Initialize(SteamVR_Input_Sources handType, Hand otherHand, bool isIV
 			{
 				throw new ArgumentException("handType must be LeftHand or RightHand");
 			}
-			bool isRightHand = handType == SteamVR_Input_Sources.RightHand;
+			isRightHand = handType == SteamVR_Input_Sources.RightHand;
 
 			// load hand profile
-			profile = HandProfileManager.Instance.GetProfile(isIVA);
+			HandProfileManager handSetting = HandProfileManager.Instance;
 
-			GripOffset = profile.gripOffset;
+			#region Setup tracking skeleton
 
-			// load hand prefab assets
-			string prefabName = isRightHand ? profile.PrefabNameRight : profile.PrefabNameLeft;
-			GameObject handPrefab = AssetLoader.Instance.GetGameObject(prefabName);
-			if (handPrefab == null)
-			{
-				throw new MissingReferenceException($"Could not load prefab: {prefabName}");
-			}
+			GameObject trackingSkeleton = AssetLoader.Instance.GetGameObject(
+				isRightHand ? 
+				handSetting.skeletonPrefabNameRight : 
+				handSetting.skeletonPrefabNameLeft);
 
-			// make instance object out of the hand prefab
-			handObject = Instantiate(handPrefab);
-			if (handObject == null)
+			if (trackingSkeleton == null)
 			{
-				throw new Exception("Could not Instantiate prefab for " + handType);
+				throw new MissingReferenceException($"Could not load tracking skeleton");
```

Comment:
include the missing name here?  I'm fine with doing that after the merge though

Reply:
Fixed by [c35e6bc](https://github.com/JonnyOThan/Kerbal-VR/pull/172/commits/c35e6bc15153e8ec6391f1a7d6308ce51420088a)